pkg/validation/limits.go

```diff
@@ -205,6 +205,9 @@ type Limits struct {
 	BloomCompactorMaxBlockSize flagext.ByteSize `yaml:"bloom_compactor_max_block_size" json:"bloom_compactor_max_block_size" category:"experimental"`
 	BloomCompactorMaxBloomSize flagext.ByteSize `yaml:"bloom_compactor_max_bloom_size" json:"bloom_compactor_max_bloom_size" category:"experimental"`
 
+	BloomCreationEnabled             bool `yaml:"bloom_creation_enabled" json:"bloom_creation_enabled" category:"experimental"`
+	BloomSplitSeriesKeyspaceByFactor int  `yaml:"bloom_split_series_keyspace_by_factor" json:"bloom_split_series_keyspace_by_factor" category:"experimental"`
```

Comment:
nit: Personally I find the name `factor` not quite right, because it implies that it is used to multiply with something.
I would name it something with series shard size.

Reply:
I wouldn't use shard since it's not sharding at all. What about `bloom_split_series_keyspace_by: 256`. I think it reads good enough.

Note that wi'll likely replace this keyspace split by something smarter using TSDB stats soon, so I wouldn't worry too much about naming here.